sparkbox/apprentice-at
# Refactor: Install Cypress Testing and Update Apprentice Filter Tests

- installs `cypress`
- updates apprentice filtering tests to use `cypress`
- adds `npm test` script to open cypress
- adds `npm test:ci`script to run tests in ci mode with 

### Testing
- npm i
- npm start to run server
- run `npm run test`
- ensure cypress interface opens
- click on `apprenticeFilterSpec.js` file
- verify all tests are passing
- close cypress
- run `npm run test:ci`
- ensure it runs the test in CI mode
